test/test_va_api_query_subpicformat.cpp

```diff
@@ -0,0 +1,63 @@
+ /*
+ * Copyright (C) 2017 Intel Corporation. All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a
+ * copy of this software and associated documentation files (the
+ * "Software"), to deal in the Software without restriction, including
+ * without limitation the rights to use, copy, modify, merge, publish,
+ * distribute, sub license, and/or sell copies of the Software, and to
+ * permit persons to whom the Software is furnished to do so, subject to
+ * the following conditions:
+ *
+ * The above copyright notice and this permission notice (including the
+ * next paragraph) shall be included in all copies or substantial portions
+ * of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+ * OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT.
+ * IN NO EVENT SHALL PRECISION INSIGHT AND/OR ITS SUPPLIERS BE LIABLE FOR
+ * ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+#include "test_va_api_query_subpicformat.h"
+#include<functional>
+
+namespace VAAPI {
+VAAPIQuerySubPicImgFmt::VAAPIQuerySubPicImgFmt()
+{
+    m_vaDisplay = doInitialize();
+}
+
+VAAPIQuerySubPicImgFmt::~VAAPIQuerySubPicImgFmt()
+{
+    doTerminate();
+}
+
+TEST_P(VAAPIQuerySubPicImgFmt, CheckSubPicImgFmt)
+{
+    uint32_t subPicFmt = GetParam();
+    uint32_t result ;
+
+    doGetMaxNumSubPicImageFormats();
+
+    doQuerySubPicImageFormats();
+
+    result = doFindSubPicImageFormatInList(subPicFmt);
+    if(result == subPicFmt)
+        SUCCEED();
+    else if(!result) {
```

Comment:
In previous functions in this file, the '{' is on the next line. Please modify the rest of the else if to follow standard indention rules.

Reply:
Fixed.